### ISSUE TITLE
API bug template

### ISSUE THREAD

**Issue by rrennick** [woocommerce/woocommerce#29676]
We need to add a bug template for reporting issues related to the api to collect information that is specific to the API.

**Comment by samueljseay** [comment]
@rrennick would you mind picking this one up? I don't think Atlas has a good view on what the template should include. Please feel free to create new templates where they make sense.

**Comment by rrennick** [comment]
@samueljseay Thanks for the ping. We have 3 APIs so this would be good to implement with a dropdown to help identify which one is related to the issue.

**Comment by juliaamosova** [comment]
We looked at this issue on the team and decided to close it as `won't fix` as we no longer see the need for the separate API issue template. 

We have 2 labels right now and each of them have a number of issues with all the details about the bug:
- `focus: rest api`([issues](https://github.com/woocommerce/woocommerce/labels/focus%3A%20rest%20api))
- `focus: store api` ([issues](https://github.com/woocommerce/woocommerce/labels/focus%3A%20store%20api))

This issue has also been open over 3 years and there wasn't a strong signal during these years that we indeed need a separate template. 

Closing this issue for now but we can always reevaluate if the need arises.
